src/eee.js

```diff
@@ -129,7 +126,7 @@ class Emitter {
         const names = Array.isArray(names_) ? names_ : [names_];
         const $res = new Result();
 
-        await Promise.delay(); // yield the current event for fully async emits
+        await 1; // yield the current event for fully async emits (await is always async)
```

Comment:
Would this work the same way? I find it way clearer.

```suggestion
        await Promise.resolve(); // yield the current event for fully async emits (await is always async)
```